dataland-frontend/src/components/resources/frameworkDataSearch/euTaxonomy/EuTaxonomyForNonFinancialsDataModel.ts

```diff
@@ -0,0 +1,642 @@
+            label: "EU Taxonomy Activity Level Reporting",
+            description: "Activity Level disclosure",
+            unit: "",
+            component: "YesNoFormField",
+            evidenceDesired: false,
+            required: false,
+            showIf: (): boolean => true,
+            certificateRequiredIfYes: false,
+          },
+          {
+            name: "numberOfEmployees",
+            label: "Number Of Employees",
+            description: "Total number of employees (including temporary workers)",
+            unit: "",
+            component: "NumberFormField",
+            evidenceDesired: false,
+            required: false,
+            showIf: (): boolean => true,
+            validation: "min:0",
+          },
+          {
+            name: "nfrdMandatory",
+            label: "NFRD Mandatory",
+            description: "The reporting obligation for companies whose number of employees is greater or equal to 500",
+            unit: "",
+            component: "YesNoFormField",
+            evidenceDesired: false,
+            required: false,
+            showIf: (): boolean => true,
+            certificateRequiredIfYes: false,
+          },
+          {
+            name: "assurance",
+            label: "Assurance",
+            description:
+              "Level of Assurance of the EU Taxonomy disclosure (Reasonable Assurance, Limited Assurance, None)",
+            unit: "",
+            component: "AssuranceFormField",
+            evidenceDesired: true,
+            required: false,
+            showIf: (): boolean => true,
+          },
+        ],
+      },
+    ],
+  },
+  {
+    name: "revenue",
+    label: "Revenue",
+    color: "orange",
+    showIf: (): boolean => true,
+    subcategories: [
+      {
+        name: "revenue",
+        label: "Revenue",
+        fields: [
+          {
+            name: "totalAmount",
+            label: "Total Revenue",
+            description:
+              "Total Revenue for the financial year. I.e. income arising in the course of an entity's ordinary activities., the amounts derived from the sale of products and the provision of services after deducting sales rebates and value added tax and other taxes directly linked to turnover. Overall turnover is equivalent to a firm's total revenues over some period of time",
+            unit: "",
+            component: "DataPointFormField",
+            evidenceDesired: true,
+            options: getDataset(DropdownDatasetIdentifier.CurrencyCodes),
+            required: false,
+            showIf: (): boolean => true,
+          },
+          {
+            name: "eligibleShare",
+            label: "Total Eligible Revenue",
+            description:
+              "Absolute value and share of the revenue where the economic activity meets taxonomy criteria for substantial contribution to climate change mitigation and does no serious harm to the other environmental objectives (DNSH criteria)",
```

Comment:
Is this tooltip correct (I thought this was the definition of Aligned Revenue)? Why only Climate Change Mitigation. We have a lot more substatnial contribution criteria.

Reply:
Will check again after 2307 is merged

Reply:
The description did not change, the current message is correct according to the latest version of the data dictionary.

Reply:
I will bring this to Julia's attention.